documentation/rosetta-modelling-component.md

```diff
@@ -1618,26 +1634,24 @@ Conditional mappings allow to build more complex mappings. Conditional mappings
 A set to mapping is used to set the value of an attribute to a constant value. They don\'t attempt to use any data from the input document as the value for the attribute and a synonym value must not be given. The type of the constant must be convertible to the type of the attribute. The constant value can be given as a string (converted as necessary) or an enum - e.g. :
 
 ```
-period PeriodEnum (1..1)
-  [synonym ISDA_Create_1_0 set to PeriodEnum.D]
-itemName string (1..1) <"In this ....">;
-  [synonym DTCC_11_0 set to "comment"]
+engineEnum EngineEnum (0..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_13 set to EngineEnum->HYBRID]
 ```
 
 A set to mapping can be conditional on a [when clause](#when-clause) - e.g.:
 
 ```
-itemName string (1..1) <"In this ....">;
-  [synonym DTCC_11_0 set to "comment" when path = "PartyWorkflowFields.comment"]
+engineEnum EngineEnum (0..1)
+  [synonym CONDITIONAL_SET_TO_EXAMPLE_13 set to EngineEnum->HYBRID when "alternativeFuelDetail" exists and "fuelDetail" exists]
```

Comment:
same comment about hybrid (not in capital letters)

Reply:
Are you not covering the "set when" mapping syntax instead of the "set" syntax? It looks like the case (see example with the "set when"). I was expecting more a attribute =value.

Reply:
not so much, this is because none of the examples made were exactly similar to the originals in the documentation and the ones that were more similar are the set-to not the set-when. I explain below the exact reasons and the proposal for resolving that.